estate/__manifest__.py

```diff
@@ -15,5 +15,10 @@
         "views/estate_property_views.xml",
         "views/estate_menus.xml",
         "views/res_users_views.xml",
+        "data/estate.property.type.csv",
+    ],
+    "demo": [
+        "demo/demo_data_estate_property.xml",
+        "demo/demo_data_estate_property_offer.xml",
```

Comment:
Please rename the files. Check guidelines for file naming https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#15file-naming the word "demo" / "data" / "views" is a suffix. And the word `data` is a suffix for files in the "data" section.

```suggestion
        "demo/estate_property_demo.xml",
        "demo/estate_property_offer_demo.xml",
```